bpf/kmesh/workload/include/workload_common.h

```diff
@@ -26,20 +26,6 @@ typedef enum {
     LB_POLICY_FAILOVER = 2,
 } lb_policy_t;
 
-union v6addr {
-    struct {
-        __u32 p1;
-        __u32 p2;
-        __u32 p3;
-        __u32 p4;
-    };
-    struct {
-        __u64 d1;
-        __u64 d2;
-    };
-    __u8 addr[16];
-} __packed;
```

Comment:
@weli-l After bump, unnamed union is not supported, so i removed it, Can you take a look